ansible/_lint_roles.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Could the lint script autogenerate that file rather than hoping we remember to update it?
It would be more robust to check all roles by default, and if needed add some to an ignore-list, than not checking roles by default.

We would need similar logic to enumerate all playbooks (`*.yml`, `host_playbooks/*.yml`, and an ignore list) anyway, so it doesn't seem overkill.

Reply:
Yes sure but for roles this is a little bit more complicated than for playbooks. Since we sometimes organize roles in subfolders we would need some heuristic to know whether a given sub directory actually contains a role or not. Probably checking for a `task` directory and a `main.yml` in there. Also we should then forbid a case where a folder is a role and has sub-roles inside - which i wouldn't want to have anyway.

Also we probably also want to have a blacklist for roles that shouldn't be checked...

Reply:
Yeah, OK; let's table that for a later PR.